content/pt/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Padrão Operador
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Operadores são extensões de software para o Kubernetes que
+fazem uso de [*recursos personalizados*](/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+para gerir aplicações e os seus componentes. Operadores seguem os  
+princípios do Kubernetes, notavelmente o [ciclo de controle](/docs/concepts/#kubernetes-control-plane).
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Motivação
+
+O padrão Operador tem como objetivo capturar o principal objetivo de um operador
+humano que gere um serviço ou um conjunto de serviços. Operadores humanos 
+responsáveis por aplicações e serviços específicos têm um conhecimento
+profundo da forma como o sistema é suposto se comportar, como é instalado
+e como deve reagir na ocorrência de problemas.
+
+As pessoas que correm cargas de trabalho no Kubernetes habitualmente gostam
+de usar automação para cuidar de tarefas repetitivas. O padrão Operador captura
+a forma como pode escrever código para automatizar uma tarefa para além do que
+o Kubernetes fornece.
+
+## Operadores no Kubernetes
+
+O Kubernetes é desenhado para automação. *Out of the box*, você tem bastante
+automação embutida no núcleo do Kubernetes. Pode usar
+o Kubernetes para automatizar instalações e executar cargas de trabalho,
+e pode ainda automatizar a forma como o Kubernetes faz isso.
+
+O conceito de {{< glossary_tooltip text="controlador" term_id="controller" >}} no
+Kubernetes permite a extensão do comportamento sem modificar o código do próprio
+Kubernetes.
+Operadores são clientes da API do Kubernetes que atuam como controladores para
+um dado [*Custom Resource*](/docs/concepts/api-extension/custom-resources/)
+
+## Exemplo de um Operador {#example}
```

Comment:
nit:
```suggestion
## Exemplo de um Operador {#exemplo}
```
?

Reply:
I don't think anchors should be translated.

Reply:
@joao-luna-98 Most anchors are translated anyway, because Hugo automatically uses the heading text. English anchors would be the exception.